healthcareai/trained_models/trained_supervised_model.py

```diff
@@ -183,17 +188,37 @@ def prepare_and_subset(self, dataframe):
             pandas.core.frame.DataFrame: A dataframe that has been run through the pipeline and subsetted to only the columns the model expects.
         """
 
+        # Add response column if not included in prediction data
```

Comment:
Please clarify comment - add prediction column
...and this (lines 191-193) should probably be it's own function that we could easily test.